src/PrestaShopBundle/Translation/Provider/TranslationFinderTrait.php

```diff
@@ -54,6 +54,10 @@ public function getCatalogueFromPaths($paths, $locale, $pattern = null)
         }
 
         foreach ($translationFiles as $file) {
+            if ('index.php' === $file->getFileName()) {
```

Comment:
Did you know that you could directly exclude this files from the finder execution ?

http://symfony.com/doc/current/components/finder.html#file-name

You could have written
```php
$translationFiles = $finder->files()->notName('index.php')->in($paths);
```